packages/webdriverio/src/commands/browser/getCookies.js

```diff
@@ -25,8 +25,8 @@
  * </example>
  *
  * @alias browser.getCookies
- * @param {String[]=} names  names of requested cookies
- * @return {Object[]}        requested cookies if existing
+ * @param {String[]=|String=} names  names of requested cookies (if omitted, all cookies will be returned)
```

Comment:
deleteCookies might need the same changes

Reply:
true, adjusted it